frontend/src/features/apartment/ApartmentNewSalePage/LoadedApartmentSalesPage.tsx

```diff
@@ -250,7 +327,7 @@ const LoadedApartmentSalesPage = ({apartment}: {apartment: IApartmentDetails}) =
                 isLoading={isCreateSaleLoading}
                 isVisible={isWarningModalVisible}
                 setIsVisible={setIsWarningModalVisible}
-                modalText="Kauppahinta ylittää laskelman enimmäishinnan. Haluatko varmasti tallentaa kaupan?"
+                modalText={`${warningMessage} Haluatko varmasti tallentaa kaupan?`}
```

Comment:
This would probably look nicer with a line break in between the `warningMessage` and "Haluatko varmasti tallentaa kaupan?"-string.